### DISCUSSION TITLE
Multiple applications on same port

### DISCUSSION THREAD

**Discussion by StallionV** [nrwl/nx#7441] [General]
Hello,

How can we run multiple applications on the same port just different base URL?
Assume we do not have access to backend since API calls are external

We are using next js and adding basePath in next.config.js will render app on localhost/basepath but ports are still different
so instead of
app1 on localhost:4200
app2 on localhost:4300
We want
app1 on localhost:4200/app1
app2 on localhost:4200/app2 running in parallel.

**Comment by pstepniewskiconfigit** [comment]
Just run nginx container with reverse proxy config

https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/
